thirdeye/thirdeye-pinot/src/test/java/com/linkedin/thirdeye/tools/CloneFunctionAndAutoTuneAlertFilterTool.java

```diff
@@ -89,33 +112,33 @@ public String getAlertFilterEvaluation(String functionId, String startTimeISO, S
   }
 
 
-  public void writeMapToCSV(Map<String, String> map, String fileName) throws IOException {
+  public void writeMapToCSV(Map<Object, Object> map, String fileName) throws IOException {
```

Comment:
Try to avoid using Map<Object, Object>. Do you need to insert objects rather than String?